src/uct/ib/ud/base/ud_ep.c

```diff
@@ -569,18 +591,12 @@ ucs_status_t uct_ud_ep_create_connected_common(const uct_ep_params_t *ep_params,
         goto out_set_ep;
     }
 
-    params.field_mask = UCT_EP_PARAM_FIELD_IFACE |
-                        UCT_EP_PARAM_FIELD_PATH_INDEX;
-    params.iface      = &iface->super.super.super;
-    params.path_index = path_index;
-
-    status = uct_ep_create(&params, &new_ep_h);
+    status = uct_ud_ep_create(iface, path_index, conn_sn, &ep);
     if (status != UCS_OK) {
         goto out;
     }
 
-    ep          = ucs_derived_of(new_ep_h, uct_ud_ep_t);
-    ep->conn_sn = conn_sn;
+    *new_ep_p = &ep->super.super;
```

Comment:
already in line 633

Reply:
removed, and force-pushed (PR is small enough)